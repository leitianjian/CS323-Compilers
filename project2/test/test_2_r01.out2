Error type 1 at Line 5: x3 variable is used without definition
Error type 7 at Line 5: unsupported type of operands for arithmatic operator
Error type 5 at Line 5: unmatching types on both sides of assignment operator (=)
