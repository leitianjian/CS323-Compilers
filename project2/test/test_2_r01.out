Error type 1 at line 5: x3 variable is used without definition
